beast/tests/test_regresscheck.py

```diff
@@ -1069,8 +844,150 @@ def test_star_type_probability_no_Av(self):
         # compare to new table
         compare_tables(expected_star_prob, Table(star_prob))
 
+    # ###################################################################
+    # tools.run tests
+
+    @pytest.mark.usefixtures("setup_create_physicsmodel")
```

Comment:
Can you say what this decorator (if I am naming this correctly) is doing?  I'm asking as I want to learn!

Reply:
I found this page (plus some random googling here and there) to be really helpful:
https://pythontesting.net/framework/pytest/pytest-fixtures-nuts-bolts/

Basically, it says to run `setup_create_physicsmodel` before and/or after running this test.  Within `setup_create_physicsmodel`, there's some setup (well, not in this particular one, but there is for `setup_create_obsmodel`), then `yield` sends it back to the test, and everything after `yield` is done after the test finishes (whether gracefully or not).  It's pretty cool!

Reply:
That is pretty cool.  Thanks for the info/education.